integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -42,51 +46,176 @@ def get_context_data(self, **kwargs):
+        }
+
+    def get_unread_feedback_context(self):
+        r"""
+        Extend context by info on unread feedback
+
+        :return: Dictionary containing the context for unreviewed pages
+        :rtype: dict
+        """
+        unread_feedback = Feedback.objects.filter(read_by=None)
+        return {
+            "unread_feedback": unread_feedback,
+        }
+
+    def get_low_hix_value_context(self):
+        r"""
+        Extend context by info on pages with low hix value
+
+        :return: Dictionary containing the context for pages with low hix value
+        :rtype: dict
         """
-        Extend context by HIX info
+        pages_under_hix_threshold = []
+        if settings.TEXTLAB_API_ENABLED and self.request.region.hix_enabled:
+            hix_pages = self.request.region.get_pages(
+                return_unrestricted_queryset=True
+            ).filter(hix_ignore=False)
+            hix_translations = PageTranslation.objects.filter(
+                language__slug__in=settings.TEXTLAB_API_LANGUAGES, page__in=hix_pages
+            ).distinct("page_id", "language_id")
+
+            for hix_translation in hix_translations:
+                if (
+                    hix_translation.hix_score is not None
+                    and hix_translation.hix_score < settings.HIX_REQUIRED_FOR_MT
+                ):
+                    pages_under_hix_threshold.append(hix_translation)
+
+            worst_page = (
+                hix_translations.order_by("hix_score").distinct().first()
+                if hix_translations
+                else None
+            )
+
+            return {
+                "pages_under_hix_threshold": pages_under_hix_threshold,
+                "worst_page": worst_page,
+                "hix_score": round(worst_page.hix_score, 2),
+            }
+        return {}
+
+    def get_outdated_pages_context(self):
+        r"""
+        Extend context by info on outdated pages
 
-        :return: The HIX context dictionary
+        :return: Dictionary containing the context for outdated pages
         :rtype: dict
         """
-        if not settings.TEXTLAB_API_ENABLED:
-            return {}
+        OUTDATED_THRESHOLD_DAYS = 365
+        OUTDATED_THRESHOLD_DATE = (
+            datetime.now() - relativedelta(days=OUTDATED_THRESHOLD_DAYS)
+        )
+
+        outdated_pages = (
+            PageTranslation.objects
+                .filter(
+                    last_updated__lte=OUTDATED_THRESHOLD_DATE.date(),
+                    page__explicitly_archived=False,
+                    language__slug=self.request.region.default_language.slug)
+                .order_by("page__id", "language__id", "-version", "last_updated")
+                .distinct("page__id", "language__id")
+                .all()
+        )
+        print(OUTDATED_THRESHOLD_DATE, outdated_pages[0].last_updated)
+        for p in outdated_pages:
+            # TODO: why does last_updated sometimes differ??
```

Comment:
@david-venhoff This is the line we spoke about on Tuesday. If you checkout this branch and go to the dashboard view, you'll see in brackets the amount of pages that fit the criteria of being outdated (being older than 365 days). If you filter for the same criteria in the page tree (or click on the title link "outdated page") you might notice, that the amount of pages differ. In my case for 1088 pages the dashboard shows 24 outdated pages in the dashboard, but only 13 in the filtered page tree view